metrics/exporter.go

```diff
@@ -137,7 +136,12 @@ func setCurMetricsExporterAndConfig(e view.Exporter, c *metricsConfig) {
 	metricsMux.Lock()
 	defer metricsMux.Unlock()
 	view.RegisterExporter(e)
-	view.SetReportingPeriod(60 * time.Second)
+	if c != nil {
+		view.SetReportingPeriod(c.reportingPeriod)
```

Comment:
Could `c.reportingPeriod` be `nil`?

Reply:
c.reportingPeriod is time.Duration (type Duration int64) - so the default value is 0 and setting reporting period to 0 is same as leaving it as its default value in Opencensus.